coreos-xhyve-run

```diff
@@ -1,4 +1,18 @@
 #!/bin/bash
+OPTIND=1
+while getopts ":f:" opt; do
+  case ${opt} in
+    f)
+      [[ -r "${OPTARG}" ]] && source "${OPTARG}" || \
+        echo "Warning: couldn't read \"${OPTARG}\". Ignoring." >&2
```

Comment:
@bakins i opted for not making it fatal for 'politeness'. matter of taste i suppose. up to you :smile:

meanwhile just rebased previous PR against upstream.